romana-install/group_vars/aws

```diff
@@ -7,5 +7,18 @@ instance_type:
     devstack: "t2.small"
     kubernetes: "t2.small"
 
-controller_image: "ami-bf3ec1fb" # Ubuntu 14.04.3 LTS
-compute_image: "ami-bf3ec1fb" # Ubuntu 14.04.3 LTS
+distro_info:
+  "ubuntu":
+    ami:  "ami-bf3ec1fb" # Ubuntu 14.04.3 LTS
+    user: "ubuntu"
+    bootstrap:
```

Comment:
Yup. We need to define this somewhere, and I couldn't think of a better spot.
It's basically normalizing the hosts before we can run ansible on them.
There's those tiny differences between distros to get to that point.